projects/angular-showcase/src/app/business/business-examples/links-showcase/links.component.html

```diff
@@ -0,0 +1,57 @@
+<div class="container-fluid">
```

Comment:
Please split up the example in small examples with as less html as possible. see select showcase as an example.

Reply:
If you like, do the same in public component :)